pkg/controller/postgres/postgres_controller.go

```diff
@@ -367,3 +378,29 @@ func getCommand(containers []*contrail.Container, containerName string) []string
 
 	return c.Command
 }
+
+func (r *ReconcilePostgres) ensureServiceExists(postgres *contrail.Postgres) (*core.Service, error) {
+	postgresService := newPostgresService(postgres)
+	_, err := controllerutil.CreateOrUpdate(context.Background(), r.client, postgresService, func() error {
+		postgresService.Spec.Ports = []core.ServicePort{
+			{Port: 5432, Protocol: "TCP"},
+		}
+		postgresService.Spec.Selector = map[string]string{"app": postgres.Name}
```

Comment:
Maybe we could just copy label from `postgres` object. It will always match in that case